<?xml version="1.0" ?>
<sdf version="1.6">
  <model name="pickup_box"> <pose>0 0 0 0 0 0</pose> <link name="box_link">
      <inertial>
        <mass>0.5</mass>
        <inertia ixx="0.000833" ixy="0" ixz="0" iyy="0.000833" iyz="0" izz="0.000833"/>
      </inertial>
      <collision name="collision">
        <geometry>
          <box><size>0.02 0.02 0.13</size></box>
        </geometry>
      </collision>
      <visual name="visual">
        <geometry>
          <box><size>0.02 0.02 0.13</size></box>
        </geometry>
        <material>
          <script>
            <uri>file://media/materials/scripts/gazebo.material</uri>
            <name>Gazebo/Blue</name>
          </script>
        </material>
      </visual>
    </link>
  </model>
</sdf>